app/src/main/res/layout/lesson_thumbnail_image_view_test_fragment.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <LinearLayout
```

Comment:
Is a linear layout actually needed or could we just use a FrameLayout here?

Reply:
Done. Used FrameLayout.